django_react_proj/users-fe/src/components/SummingTime/SummingTime.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { API_URL } from '../../constants';
+import styles from './SummingTime.module.css';
+
+const Subtract = () => {
+  const [users, setUsers] = useState();
+  let timevalue = 0;
+  const getUsers = () => {
+    axios.get(API_URL).then((res) => setUsers(res.data));
+  };
+
+  React.useEffect(() => {
+    const handle = setInterval(getUsers, 1000);
+    return () => clearInterval(handle);
+  }, []);
+
+  if (users) {
+    users.forEach((user) => {
+      const uptime = user.deadlinev2;
```

Comment:
This whole logic can be simplified if `deadline` and any other time related entry would be kept in database as a timestamp. It's easier to work with and it's a standardize form of time  / date representation in any programming language. 


This is a current representation of time 
```js
-> new Date().getTime()
<- 1619437024346
```

`1619437024346` is a number which represents number of ms since `1/1/1970`  which means if you use number `1619437024346` you can be sure it's pointing into the specific point in time and every parser that works with this number know how to handle it . JS and Python have naively support for timestamps.

For your case it would be as simple as subtracting  `deadline` , e.g. as i'm writing this current timestamp is changed and is equal to:

`1619437242361`

which means if I subtract this one from the previous one I get the duration which took me writing this comment

`1619437242361 - 1619437024346 = 218015`

`218015` ms, you can now format it in any shape (e.g. using `date-fns` utilities functions or manually)

218015 / 1000 = 218.015s
218.015 / 60 = 3.633m  = 3m + 0.633m
0.633 m = 0.633 * 60s = 37.98s

It took me 3m and 38s to write this